src/main/scala/diplomaticobjectmodel/model/OMPorts.scala

```diff
@@ -125,15 +125,15 @@ case class SystemPort(
 
 object OMPortMaker {
   val protocolSpecifications = Map[ProtocolType, String](
-    AHBProtocol -> "AMBA 3 AHB-Lite Protocol",
-    AXI4Protocol -> "AMBA 3 AXI4-Lite Protocol",
-    APBProtocol -> "AMBA 3 APB Protocol",
+    AHBProtocol -> "AHB Protocol",
+    AXI4Protocol -> "AXI Protocol",
+    APBProtocol -> "APB Protocol",
     TLProtocol -> "TileLink specification"
   )
 
   val protocolSpecificationVersions = Map[ProtocolType, String](
-    AHBProtocol -> "1.0",
-    AXI4Protocol -> "1.0",
+    AHBProtocol -> "3",
+    AXI4Protocol -> "4",
     APBProtocol -> "1.0",
```

Comment:
why is APB still 1.0 when before it was "AMBA 3" ? Is this right @terpstra ? Do the AMBA spec version stay alighed with the specific bus interfaces...?

Reply:
According to [this wiki page](https://en.wikipedia.org/wiki/Advanced_Microcontroller_Bus_Architecture), they track more or less, but not perfectly.